File: PracticasPython/Practica7/JuegoAdivinaPar.py
from JuegoAdivinaNumero import JuegoAdivinaNumero
class JuegoAdivinaPar(JuegoAdivinaNumero):
    def __init__(self, numeroDeVidas, record, numeroAAdivinar):
        super().__init__(numeroDeVidas, record, numeroAAdivinar)
    def validaNumero(self, n):
            if 1 <= n <= 10 and n%2 == 0:
                return True
            elif 1 <= n <= 10 and n%2 != 0:
                print("ERROR")
                return False
            else:
                return False

File: PracticasJava/Practica7/src/JuegoAdivinaPar.java

public class JuegoAdivinaPar extends JuegoAdivinaNumero {
	public JuegoAdivinaPar(int numeroDeVidas, int record, int numeroAAdivinar) {
        super(numeroDeVidas, record, numeroAAdivinar);
    }
	@Override
	public boolean validaNumero(int n) {
		if (n > (-1) && n < 11 && n % 2== 0) {
			return true;
		} else if (n > (-1) && n < 11 && n % 2== 1) {
			System.out.println("ERROR");
			return false;
		} else {
            return false;  
        }
	}
}
